<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuColors">
    <property name="title">
     <string>Colors</string>
    </property>
    <addaction name="actionRed"/>
    <addaction name="actionBlue"/>
    <addaction name="actionGreen"/>
    <addaction name="actionYellow"/>
    <addaction name="actionMagenta"/>
    <addaction name="actionBlack"/>
    <addaction name="actionWhite"/>
    <addaction name="actionGray"/>
   </widget>
   <widget class="QMenu" name="menuSecond_Window">
    <property name="title">
     <string>Second Window</string>
    </property>
    <addaction name="actionSecond_Page"/>
   </widget>
   <widget class="QMenu" name="menuThird_Window">
    <property name="title">
     <string>Third Window</string>
    </property>
    <addaction name="actionThird_Page"/>
   </widget>
   <addaction name="menuColors"/>
   <addaction name="menuSecond_Window"/>
   <addaction name="menuThird_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionYellow">
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionMagenta">
   <property name="text">
    <string>Magenta</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionThird_Page">
   <property name="text">
    <string>Third Page</string>
   </property>
  </action>
  <action name="actionSecond_Page">
   <property name="text">
    <string>Second Page</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
